tox.ini

```diff
@@ -71,9 +71,13 @@ deps =
     lib: ops
     charm: responses==0.20.0
     charm: httpcore==0.14.7
+    unit: {[testenv:unit]deps}
+    integration: {[testenv:integration]deps}
 commands =
     charm: mypy {[vars]src_path} {posargs}
     lib: mypy --python-version 3.5 {[vars]lib_path} {posargs}
+    unit: mypy {[vars]tst_path}/unit {posargs}
+    integration: mypy {[vars]tst_path}/integration {posargs}
```

Comment:
Why are these added back?

Reply:
I think those were removed by mistake. The correct change is
```diff
-envlist = lint, static-{charm,lib,unit,integration}, unit
+envlist = lint, static-{charm,lib}, unit
```
I've done it in this PR here.

As for the envs themselves - I use them! :)